libraries/AP_Scripting/generator/description/bindings.desc

```diff
@@ -205,3 +205,21 @@ singleton AP_Mission method get_prev_nav_cmd_id uint16_t
 singleton AP_Mission method get_current_nav_id uint16_t
 singleton AP_Mission method get_current_do_cmd_id uint16_t
 singleton AP_Mission method num_commands uint16_t
+singleton AP_Mission method get_item boolean uint16_t 0 UINT16_MAX mavlink_mission_item_int_t'Null
+singleton AP_Mission method set_item boolean uint16_t 0 UINT16_MAX mavlink_mission_item_int_t
+
+userdata mavlink_mission_item_int_t field param1 float read write -FLT_MAX FLT_MAX
+userdata mavlink_mission_item_int_t field param2 float read write -FLT_MAX FLT_MAX
+userdata mavlink_mission_item_int_t field param3 float read write -FLT_MAX FLT_MAX
+userdata mavlink_mission_item_int_t field param4 float read write -FLT_MAX FLT_MAX
+userdata mavlink_mission_item_int_t field x int32_t read write -INT32_MAX INT32_MAX
+userdata mavlink_mission_item_int_t field y int32_t read write -INT32_MAX INT32_MAX
+userdata mavlink_mission_item_int_t field z float read write -FLT_MAX FLT_MAX
+userdata mavlink_mission_item_int_t field seq uint16_t read write 0 UINT16_MAX
+userdata mavlink_mission_item_int_t field command uint16_t read write 0 UINT16_MAX
+-- userdata mavlink_mission_item_int_t field target_system uint8_t read write 0 UINT8_MAX
```

Comment:
I guess these are comment out lines?  normally we don't leave comments out lines in the code..

Reply:
in this particular instance they are here as a self-documenting reminder that one-day these other attributed might be useful, but right now they aren't used.